README.md

```diff
@@ -1094,6 +1095,89 @@ Server: spray-can/1.3.3
 }
 ```
 
+## GET /workflows/:version/:id/metadata
```

Comment:
Sorry I'm not sure what you're looking for here, could you sketch what you mean?